### DISCUSSION TITLE
manage turtle/polite queue with variables

### DISCUSSION THREAD

**Discussion by ouafnico** [Mailu/Mailu#3075] [User support]
Hello

I'm actually using turtle queue from v2.0 to slow some mails for orange.

I need to slow more than turtle configuration.

Can it be possible in future version to manage the options of the queue, with some variables ? 
Or it force to override the all master.cf

Thanks

**Comment by nextgens** [comment]
We can definitely make turtle go even slower. How slow are we talking about here? Do you have a master.cf that works?

**Reply by ouafnico** [reply]
I guess we need something like this, more than turtle...

```
# Snail policy
snail      unix  -       -       n       -       -       smtp
  -o syslog_name=postfix-snail
  -o snail_destination_concurrency_limit=1
  -o snail_destination_rate_delay=1
  -o snail_destination_recipient_limit=1
  -o snail_destination_concurrency_failed_cohort_limit=1
```

**Reply by nextgens** [reply]
I have made a PR for it... next release will use a slower default #3076
